tests/10-issue.js

```diff
@@ -26,95 +26,325 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
+              if(issuedVc.credentialStatus.statusSize) {
+                it('If present, ' +
+                  'statusSize MUST be an integer greater than zero.',
+                async function() {
+                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20statusSize%20MUST%20be%20an%20integer%20greater%20than%20zero.';
+                  issuedVc.credentialStatus.statusSize.should.be.a('number',
+                    'Expected statusSize to be an integer.');
+                  // TODO test for integer
+                  assert(Number.isInteger(
+                    issuedVc.credentialStatus.statusSize),
+                  'Expected statusSize to be an integer.');
+                  issuedVc.credentialStatus.statusSize.should.be.gt(0,
+                    'Expected statusSize to be >= 0.');
+                });
+                if(issuedVc.credentialStatus.statusSize.gt(1)) {
+                  it('If statusSize is provided and is greater than 1, ' +
+                    'then the property credentialStatus.statusMessage ' +
+                    'MUST be present',
+                  async function() {
+                    this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20statusSize%20is%20provided%20and%20is%20greater%20than%201%2C%20then%20the%20property%20credentialStatus.statusMessage%20MUST%20be%20present';
+                    issuedVc.credentialStatus.should.have.own.property(
+                      'statusMessage');
+                  });
+                }
+                it('The number of status messages MUST equal the number ' +
+                  'of possible values.',
+                async function() {
+                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=the%20number%20of%20status%20messages%20MUST%20equal%20the%20number%20of%20possible%20values';
+                  issuedVc.credentialStatus.statusSize.should.be.an('array');
+                  issuedVc.credentialStatus.statusSize.length.should.be.equal(
+                    issuedVc.credentialStatus.statusSize);
+                });
+                it('If present, the statusMessage property MUST be an array, ' +
+                  'the length of which MUST equal the number of possible ' +
+                  'status messages indicated by statusSize',
+                async function() {
+                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20the%20statusMessage%20property%20MUST%20be%20an%20array%2C%20the%20length%20of%20which%20MUST%20equal%20the%20number%20of%20possible%20status%20messages%20indicated%20by%20statusSize';
+                  issuedVc.credentialStatus.statusSize.should.be.an('array');
+                  issuedVc.credentialStatus.statusSize.length.should.be.equal(
+                    issuedVc.credentialStatus.statusSize);
```

Comment:
a reason here as the second param to equal would help, but feel free to ignore if low on time.

Reply:
added a reason for the test failure